<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QnUserManagementWidget</class>
 <widget class="QWidget" name="QnUserManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout" stretch="0,0">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
     <item>
      <widget class="QLineEdit" name="filterLineEdit">
       <property name="placeholderText">
        <string>Search users</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="nx::client::desktop::TreeView" name="usersTable">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <attribute name="headerStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_1">
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QPushButton" name="enableSelectedButton">
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disableSelectedButton">
         <property name="text">
          <string>Disable</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteSelectedButton">
         <property name="text">
          <string>Delete...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="clearSelectionButton">
         <property name="text">
          <string>Clear Selection</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="createUserButton">
       <property name="text">
        <string>New User...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rolesButton">
       <property name="text">
        <string>Edit Roles...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ldapSettingsButton">
       <property name="text">
        <string>LDAP Settings...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fetchButton">
       <property name="text">
        <string>Fetch Users from LDAP...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::client::desktop::TreeView</class>
   <extends>QTreeView</extends>
   <header location="global">nx/client/desktop/common/widgets/tree_view.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterLineEdit</tabstop>
  <tabstop>usersTable</tabstop>
  <tabstop>enableSelectedButton</tabstop>
  <tabstop>disableSelectedButton</tabstop>
  <tabstop>deleteSelectedButton</tabstop>
  <tabstop>clearSelectionButton</tabstop>
  <tabstop>createUserButton</tabstop>
  <tabstop>rolesButton</tabstop>
  <tabstop>ldapSettingsButton</tabstop>
  <tabstop>fetchButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
